packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx

```diff
@@ -187,16 +202,35 @@ export default function usePlanBillingDescription( {
 					);
 				}
 
+				/* If the offer is for X years of a yearly plan */
 				if ( 'year' === introOffer.intervalUnit ) {
+					if ( 1 === introOffer.intervalCount ) {
+						// Note: Leaving out originalPriceFullTermText to
+						// match the existing text but we should probably
+						// include it in the future. See
+						// https://github.com/Automattic/wp-calypso/pull/86434
```

Comment:
Not sure I follow why this is necessary? Can we not leave it the same it was before?

```
return translate(
						'per month, for your first %(introOfferIntervalUnit)s,{{br/}}' +
							'then %(rawPrice)s billed annually, excl. taxes',
						{
							args: {
								rawPrice: originalPriceFullTermText,
								introOfferIntervalUnit: introOffer.intervalUnit,
							},
							components: { br: <br /> },
							comment: 'excl. taxes is short for excluding taxes',
						}
					);
```